src/utils/cardUtilityFunctions.js

```diff
@@ -72,25 +74,49 @@ export const compareGrains = (grain1, grain2) => {
 export const determineMaxValueCardAttributeCount = (size, currentAttributeCount) => {
   let attributeCount = currentAttributeCount;
   switch (size) {
-    case CARD_SIZES.XSMALL:
+    case CARD_SIZES.SMALL:
       attributeCount = 1;
       break;
-    case CARD_SIZES.XSMALLWIDE:
+    case CARD_SIZES.SMALLWIDE:
       attributeCount = 2;
       break;
     case CARD_SIZES.MEDIUM:
-    case CARD_SIZES.SMALL:
-    case CARD_SIZES.WIDE:
+    case CARD_SIZES.MEDIUMTHIN:
+    case CARD_SIZES.MEDIUMWIDE:
       attributeCount = 3;
       break;
     case CARD_SIZES.LARGE:
       attributeCount = 5;
       break;
-    case CARD_SIZES.TALL:
-    case CARD_SIZES.XLARGE:
+    case CARD_SIZES.LARGETHIN:
+    case CARD_SIZES.LARGEWIDE:
       attributeCount = 7;
       break;
     default:
   }
   return attributeCount;
 };
+
+export const getUpdatedCardSize = oldSize => {
+  const changedSize =
+    oldSize === 'XSMALL'
+      ? 'SMALL'
+      : oldSize === 'XSMALLWIDE'
+      ? 'SMALLWIDE'
+      : oldSize === 'WIDE'
+      ? 'MEDIUMWIDE'
+      : oldSize === 'TALL'
+      ? 'LARGETHIN'
+      : oldSize === 'XLARGE'
+      ? 'LARGEWIDE'
+      : null;
+  let newSize = oldSize;
+  if (changedSize) {
+    warning(
```

Comment:
`warning()` should only be called in a development environment. You can use the `__DEV__` variable to conditionally call the function.